app-misc/gcstar/gcstar-1.7.3.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# GUIDE:
+# - find dependencies:
+#   # ebuild gcstar-1.7.3.ebuild install
+# - find optional components, useful to determine IUSE flags:
+#   # grep checkModule /.../work/* -R
+#   # grep checkOptionalModule /.../work/* -R
+# - find all modules:
+#   # grep -E 'use .+;' /.../work/* -R
+
+# TODO:
+# - missing gtkspell:3 perl binding (dev-perl/gtk2-spell is masked)
+
+EAPI=7
+
+inherit desktop xdg-utils
+
+DESCRIPTION="Manage your collections of movies, games, books, music and more"
+HOMEPAGE="http://www.gcstar.org/"
```

Comment:
Since this is essentially a new upstream, `HOMEPAGE` should additionally contain `https://gitlab.com/Kerenoc/GCstar`

Reply:
Thanks.